### DISCUSSION TITLE
Exporting layers as seperate images

### DISCUSSION THREAD

**Discussion by NordicS** [Orama-Interactive/Pixelorama#773] [Ideas]
I feel like this would be a rather useful feature to add. Especially for making parallax scenes and exporting character limbs/pieces for skeletal animations. While currently, you are able to export layers by hiding the layers you don't want to export and exporting layers one by one and renaming each picture, but that is very time-consuming. 
The current process for exporting layers is: hide layer(s), reveal layer(s), open exporter, rename file, export. Repeat the process for each layer you want to export. 
The exporter could look at the layer name and naming the exported image with the layer's name.

**Comment by IcedQuinn** [comment]
I want to come in and +1 this.

I recently did sprite work for a game where the sprite must be kept across many layers. Exporting this requires painfully going to each layer and doing a manual export for each one.

Especially one sprite where many iterations with colors were requested. I wanted to move the sprite over to Krita (where i can use non-destructive tools to test different designs after the pixel and lighting is done in Pixelorama) but this process is entirely too manual when your game requires 12 layers to be separately exported for an asset.

**Comment by OverloadedOrama** [comment]
Implemented in 90f7df0f5d68413a84ed6cd24eb0636661f653c4 and it will be a part of update 1.0.
